service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/cmo/DraftsOrdersUploadedEventHandlerTest.java

```diff
@@ -171,6 +184,108 @@ void shouldNotSendEmailIfNoJudgeEmail() {
         verifyNoInteractions(notificationService);
     }
 
+    @Test
+    void shouldNotSendNotificationForAdminWhenNoOrdersPresent() {
+        CaseData caseData = CaseData.builder().build();
+
+        DraftOrdersUploaded event = new DraftOrdersUploaded(caseData);
+        eventHandler.sendNotificationToAdmin(event);
+        verifyNoInteractions(notificationService);
+        verifyNoInteractions(agreedCMOContentProvider);
+    }
+
+    @Test
+    void shouldNotSendNotificationForAdminWhenNoAgreedCMOPresent() {
+        final Element<HearingBooking> hearing = hearingWithJudgeEmail("judge1@test.com");
+        final HearingOrdersBundle bundle = ordersBundle(hearing.getId(), DRAFT_CMO, C21);
+
+        CaseData caseData = CaseData.builder()
+            .hearingDetails(List.of(hearing))
+            .hearingOrdersBundlesDrafts(wrapElements(bundle))
+            .build();
+
+        DraftOrdersUploaded event = new DraftOrdersUploaded(caseData);
+        eventHandler.sendNotificationToAdmin(event);
+        verifyNoInteractions(notificationService);
+        verifyNoInteractions(agreedCMOContentProvider);
+    }
+
+    @Test
+    void shouldSendNotificationForAdminWhenTemporaryHearingJudgeAssigned() {
```

Comment:
judge email should not affect this functionality

Reply:
it is to check that hearing judge / allocated judge will be used in content provider, I think it's valid? need to provide correct judge data to agreedCMOContentProvider so that name field is correct

Reply:
yeah, though it is about checking judge email